test/functional/services/dashboard/visualizations.ts

```diff
@@ -121,5 +128,7 @@ export class DashboardVisualizationsService extends FtrService {
     await this.visEditor.setMarkdownTxt(markdown);
     await this.visEditor.clickGo();
     await this.testSubjects.click('visualizesaveAndReturnButton');
+    await this.header.waitUntilLoadingHasFinished();
+    await this.dashboard.waitForRenderComplete();
```

Comment:
Lets remove `waitForRenderComplete` after `createAndEmbedMarkdown` call at https://github.com/elastic/kibana/blob/main/test/functional/apps/dashboard/group1/create_and_add_embeddables.ts#L220

Reply:
Ahh good call - done in f646bb95848c25441f10c212678ecc062c771854 :+1: Good catch